WcaOnRails/spec/models/result_spec.rb

```diff
@@ -342,11 +342,8 @@ def result_with_n_solves(n, options)
     end
 
     it "times over 10 minutes must be rounded" do
-      result = FactoryBot.build :result, value2: 10*6000 + 4343
-      expect(result).to be_invalid_with_errors(value2: ["times over 10 minutes should be rounded"])
-
-      result.value2 = 10*6000 + 4300
```

Comment:
I have no idea why it worked before, maybe some Rails bug that has been fixed...
Reasoning:
- Here is the code responsible for validation of each solve:
https://github.com/thewca/worldcubeassociation.org/blob/7fad333b61454adec492e13b6eaa87cea1abb2b5/WcaOnRails/app/models/result.rb#L26-L33
- As you can see `solve_times` are cached like so:
https://github.com/thewca/worldcubeassociation.org/blob/7fad333b61454adec492e13b6eaa87cea1abb2b5/WcaOnRails/lib/result_methods.rb#L32-L38
- Given the above, setting attribute like `result.value2` shouldn't update `solve_times` array, which would result in the same error.

More surprisingly, when I try the same with `write_attribute :value2, 10*6000 + 4300` it passes.

Either way, I just wrote the simplest code that makes this pass as we don't wanna get rid of `solve_times` caching I think.

Reply:
Very interesting! I agree that it is very strange that this ever worked.

Ideally, I think we would try to invalidate our cache whenever an attribute changes, but I don't think that's super important to handle right now. The change you've implemented LGTM!